pkg/operator/objects.go

```diff
@@ -391,10 +391,11 @@ cat << EOF > /tmp/config/ovirt-config.yaml
 ovirt_url: $OVIRT_URL
 ovirt_username: $OVIRT_USERNAME
 ovirt_password: $OVIRT_PASSWORD
-ovirt_cafile: $OVIRT_CAFILE
+# set a valid path only if ca bundle has content
+ovirt_cafile: ${OVIRT_CA_BUNDLE+$OVIRT_CAFILE}
 ovirt_insecure: $OVIRT_INSECURE
 EOF
-echo "$OVIRT_CA_BUNDLE" > $OVIRT_CAFILE`,
+[[ $OVIRT_CAFILE ]] && echo "$OVIRT_CA_BUNDLE" > $OVIRT_CAFILE`,
```

Comment:
I think you wanted
[[ -n $OVIRT_CA_BUNDLE ]] && echo "$OVIRT_CA_BUNDLE" > $OVIRT_CAFILE`,

Reply:
it can work with both vars, but the difference is that in case OVIRT_CAFILE is empty the command will fail (because bash don't like `echo 123 > `

Reply:
but I didn't nullified OVIRT_CAFILE....